CefSharp/WebBrowserExtensions.cs

```diff
@@ -202,24 +203,21 @@ public static void ExecuteScriptAsync(this IWebBrowser browser, string methodNam
                 for (int i = 0; i < args.Length; i++)
                 {
                     var obj = args[i];
-                    if(obj == null)
+                    if (obj == null)
                     {
                         stringBuilder.Append("null");
                     }
+                    else if (numberTypes.Contains(obj.GetType()))
+                    {
+                        stringBuilder.Append(string.Format(CultureInfo.InvariantCulture, "{0}", args[i]));
+                    }
+                    else if (obj is bool)
+                    {
+                        stringBuilder.Append(args[i].ToString().ToLowerInvariant());
+                    }
                     else
                     {
-                        var encapsulateInSingleQuotes = !numberTypes.Contains(obj.GetType());
-                        if(encapsulateInSingleQuotes)
-                        {
-                            stringBuilder.Append("'");
-                        }
-
-                        stringBuilder.Append(args[i].ToString());
-
-                        if (encapsulateInSingleQuotes)
-                        {
-                            stringBuilder.Append("'");
-                        }
+                        stringBuilder.Append("'").Append(args[i].ToString().Replace("'","\\'")).Append("'");
```

Comment:
This in my opinion is getting a little hard to read, please reformat for readability :+1: